features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -190,7 +190,8 @@ repos:
 
 - name: cugraph
   path: cugraph
-  git: {<<: *git_defaults, repo: cugraph}
+  # TODO(jameslamb): revert this before merging
+  git: {<<: *git_defaults, repo: cugraph, upstream: jameslamb, tag: libcugraph-wheel}
```

Adding this suggestion so we don't forget to do this before merging.

```suggestion
  git: {<<: *git_defaults, repo: cugraph}
```